### PR TITLE
Make the jira_sync_config more KU friendly

### PR DESCRIPTION
**Pull request by addyess**
Fixes from comments:
*  https://github.com/canonical/k8s-operator/pull/84/files#diff-eee108ea49e324e51ab610c63c844d18b5b59c80bf21fd9a5b86dbecd8ad1983R12-R14

### CODE REVIEW

**Comment by addyess** [comment]
@beliaev-maksim take a look please

File: .github/.jira_sync_config.yaml [hunk 1]
```diff
@@ -10,14 +10,11 @@ settings:
...
-    - IoT
-    - DACH TT
+    - K8s charm
       
   # (Optional) GitHub labels. Only issues with one of those labels will be synchronized.
   # If not specified, all issues will be synchronized
-  labels:
-    - bug
-    - custom
+  labels: []
```
Comment (beliaev-maksim) [inline .github/.jira_sync_config.yaml:17]:
```suggestion
#  labels: []
```

File: .github/.jira_sync_config.yaml [hunk 2]
```diff
@@ -10,14 +10,11 @@ settings:
   # (Optional) Jira project components that should be attached to the created issue
   # Component names are case-sensitive
   components:
-    - IoT
-    - DACH TT
+    - K8s charm
```
Comment (beliaev-maksim) [inline .github/.jira_sync_config.yaml:13]: Please ensure that this component is added in Jira (case sensitive)

You can ping me in MM

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]
Generally, LGTM

File: .github/.jira_sync_config.yaml (hunk 2 above)
Comment (beliaev-maksim) [reply .github/.jira_sync_config.yaml:13]: done

**Review by louiseschmidtgen — APPROVED** [review]
Thank you!

**Pull request merged**